php/Terminus/Site.php

```diff
@@ -399,7 +399,7 @@ public function owner($owner=null) {
 
   public function team() {
     $method = 'GET';
-    $path = 'team';
+    $path = 'memberships/users';
     $options = array();
     $response = \Terminus_Command::request('sites', $this->getId(), $path, $method, $options);
```

Comment:
Memberships API is a paged, which means you'll need to use `Terminus_Command::paged_request`